src/org/usfirst/frc4904/sovereignty/FusedSensor.java

```diff
@@ -0,0 +1,86 @@
+package org.usfirst.frc4904.sovereignty;
+
+
+import org.usfirst.frc4904.standard.custom.sensors.InvalidSensorException;
+
+public class FusedSensor<T> implements Fusible<T> {
+	protected final Fusible<T> primarySensor;
+	protected final Fusible<T> fallbackSensor;
+	protected final Fusible<T> lastResortSensor;
+
+	public FusedSensor(T defaultValue, Fusible<T> primarySensor, Fusible<T> secondarySensor, Fusible<T>... sensors) {
+		Fusible<T> lastSensor = sensors[sensors.length - 1];
+		for (int i = sensors.length - 2; i >= 0; i--) {
+			lastSensor = new FusedSensor<>(sensors[i], lastSensor);
```

Comment:
I'm not sure I understand your comment on doing this recursively in other places, but doing this recursively would be pretty inefficient. You would have to call System.arrayCopy or Array.copyOf on each call of the constructor to shorten the sensors array. With this system, it's pretty efficient, and it saves us from having to throw errors in the constructor or anything. Maybe I'm missing something, idk.